tests/org.eclipse.cdt.lsp.clangd.tests/src/org/eclipse/cdt/lsp/clangd/tests/TestUtils.java

```diff
@@ -61,6 +61,24 @@ public static IProject createCProject(String projectName) throws CoreException {
 		return project;
 	}
 
+	//	static IProject createEmptyCMakeProject(String projectName) {
```

Comment:
Do we need this code or not? If not, let's remove it

Reply:
No, can be removed